src/OrchardCore.Modules/OrchardCore.Alias/Migrations.cs

```diff
@@ -27,13 +27,30 @@ public int Create()
             SchemaBuilder.CreateMapIndexTable(nameof(AliasPartIndex), table => table
                 .Column<string>("Alias", col => col.WithLength(AliasPartDisplayDriver.MaxAliasLength))
                 .Column<string>("ContentItemId", c => c.WithLength(26))
+                .Column<bool>("Latest", c => c.WithDefault(true))
+                .Column<bool>("Published", c => c.WithDefault(true))
```

Comment:
I would index these columns as most queries will use it. And it's safe.

Reply:
Done.

Have adjusted alias maximum size to 767, based on tests against MySql and MS SQL index key length maximums.
767 under utf8 for MySql is the largest the Alias can go based on 4 char * 767 = 3068 + 2 for the `Latest` and `Published` index keys = `3070` **the maximum key length for a MySql index.**

This is not far off the maximum for MS SQL under default collation of 1700 bytes (uses less per charactor)

Have also placed better testing against maximum lengths in the `AliasHandler`, and tested.

Will apply some of this code to the `AutorouteHandler` (without the changes to length as we do not index, not need to index the `Path` property) but in another PR